virtualizarr/backend.py

```diff
@@ -133,9 +136,41 @@ def open_virtual_dataset(
         raise NotImplementedError()
 
     # if filetype is user defined, convert to FileType
+
     if filetype is not None:
         filetype = FileType(filetype)
 
+    if filetype == FileType.kerchunk:
+        # add parquet path
+        from fsspec.implementations.reference import LazyReferenceMapper
+
+        from virtualizarr.readers.kerchunk import dataset_from_kerchunk_refs
+
+        fs = _FsspecFSFromFilepath(filepath=filepath, reader_options=reader_options)
+
+        # yuck, but JSON doesn't have magic bytes
+        if fs.filepath.endswith(".json"):
```

Comment:
we won't be able to work around detecting the `json` extension, but the first couple of bytes should be something like `{"version": 1, "refs":` (plus whitespace). This won't work for `.json.zstd`, but if we truly want to support that we can add that in a future PR?

Reply:
I agree, let's punt on the auto-detection, and just get this in.

One reason we want this feature is to help people start moving their kerchunk references into Icechunk format, so it would be good to get this in now Icechunk is out.